tests/linen/linen_linear_test.py

```diff
@@ -271,6 +271,18 @@ def test_embed(self):
     np.testing.assert_allclose(y, dummy_embedding[None])
     z = embed_module.apply(initial_params, jnp.ones((3,)), method=embed_module.attend)
     np.testing.assert_allclose(z, 3. * jnp.arange(4))
+  
+  def test_non_final_axis(self):
```

Comment:
For completeness we could also add a testcase where `axis` is a tuple, so we make sure it works on multiple axes as well.

Reply:
done